src/Symfony/Component/Security/Core/User/InMemoryUserProvider.php

```diff
@@ -24,6 +24,9 @@
  */
 class InMemoryUserProvider implements UserProviderInterface
 {
+    /**
+     * @var User[]
```

Comment:
We rarely document privates.

This one is wrong by the way. It should be `UserInterface`.